### ISSUE TITLE
Always return "caught!"

### ISSUE THREAD

**Issue by 0xFarid** [jamesstonehill/api_error_handler#21]
Thank you for good gem.

When i want to use this gem it always return "caught!" string and there isn't proper json structure.

How can i solve this problem?

**Comment by jamesstonehill** [comment]
Hi, sorry for the late reply. Can you share a stack trace?

**Comment by Mandawi** [comment]
Hi, I'm getting the same issue. I don't have a stack trace showing up. Whenever I get an error it returns 
```
caught!
```

**Comment by Mandawi** [comment]
My code looks like:
```ruby
class Api::V1::UsersController < ApplicationController
  before_action :set_user, only: %i[show edit update destroy]
  handle_api_errors(format: :json_api)

  # GET /users/:id
  def show
    render json: @user
  end

  # POST /users
  def create
    @user = User.new(user_params)
    @user.save
    render json: @user, status: :ok
  end

  # PATCH/PUT /users/:id
  def update
    @user.update(user_params)
    render json: @user, status: :ok
  end

  # DELETE /users/1
  def destroy
    @user.destroy
    render json: "", status: :ok
  end

  private

  # Use callbacks to share common setup or constraints between actions.
  def set_user
    @user = User.find(params[:id]).as_json(except: %i[id created_at updated_at encrypted_password confirmation_token 
                                                      remember_token])
  end

  # Never trust parameters from the scary internet, only allow the white list through.
  def user_params
    params.require(:user).permit(:name, :email, :password, :in_school, :major)
  end
end

```

**Comment by Babar-Livestorm** [comment]
It's a problem with the version of the gem being installed. I used this to install it and it works
```ruby
gem "api_error_handler", git: "https://github.com/jamesstonehill/api_error_handler.git", branch: "master"
```
Will install it with one of the tagged versions since `master` can be dangerous

**Comment by jamesstonehill** [comment]
I've released version 0.2.1 which should fix this issue.
